src/main/features/response/tasks/howMuchHaveYouPaidTask.ts

```diff
@@ -6,7 +6,13 @@ const validator = new Validator()
 
 export class HowMuchHaveYouPaidTask {
   static isCompleted (responseDraft: ResponseDraft): boolean {
```

Comment:
That task should only take `howMuchHaveYouPaid`. Then task list builder, depending on what task is being used, passes right instance for validation.

In the future we will remove task classes, similar to HowMuchHaveYouPaidTask, completely as they only call `!!model && validator.validateSync(model).length === 0` in most of the cases